### DISCUSSION TITLE
Ability to run single check

### DISCUSSION THREAD

**Discussion by robertmarney** [enlightn/enlightn#62] [Q&A] [answered]
Is there a command to (re)run a single check?

Use case is the check notes a failure of multi parts, after making changes it would be good to not need to run the entire suite again.

**Comment by paras-malhotra** [comment, marked as answer]
@robertmarney, you may run single checks by providing the class name as optional arguments to the Enlightn command like:

```bash
php artisan enlightn Enlightn\\Enlightn\\Analyzers\\Security\\CSRFAnalyzer
```

For more info, you can refer the docs [here](https://www.laravel-enlightn.com/docs/getting-started/usage.html#enlightn-command).

**Reply by robertmarney** [reply]
Thank you! I must be blind... :)

**Reply by paras-malhotra** [reply]
Glad it helped :)
